net/procfs/net_statistics.c

```diff
@@ -40,7 +40,7 @@
 
 #if defined(CONFIG_NET_IPv4) || defined(CONFIG_NET_IPv6) || \
     defined(CONFIG_NET_TCP) || defined(CONFIG_NET_UDP) || \
-    defined(CONFIG_NET_ICMP) || defined(CONFIG_NET_ICMPv6)
+    defined(CONFIG_NET_ICMP) || defined(CONFIG_NET_ICMPv6) || defined(CONFIG_NET_CAN)
```

Comment:
Please move defined(CONFIG_NET_CAN) to the next line to keep two per line, or change previous line to keep three per line

Reply:
Done

Reply:
let's fix the problem in the same patch. @occam25

Reply:
I guess you mean leaving the `ret = 0` fix discussed in the other comment to be part of this PR. I agree, otherwise the CAN statistics implemented in this patch would not work properly